From 36adc0000d169b6d8615730ec6ec68b029cf5e01 Mon Sep 17 00:00:00 2001
From: Fathi Boudra <fathi.boudra@linaro.org>
Date: Mon, 20 Mar 2017 12:18:20 +0200
Subject: [PATCH] selftests: lib: add config fragment for bitmap, printf and
 prime numbers tests

test_bitmap, test_printf and prime_numbers are expected to be built as
modules.

Signed-off-by: Fathi Boudra <fathi.boudra@linaro.org>
Tested-by: Naresh Kamboju <naresh.kamboju@linaro.org>
Signed-off-by: Shuah Khan <shuahkh@osg.samsung.com>
---
 tools/testing/selftests/lib/config | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 tools/testing/selftests/lib/config

diff --git a/tools/testing/selftests/lib/config b/tools/testing/selftests/lib/config
new file mode 100644
index 000000000000..126933bcc950
--- /dev/null
+++ b/tools/testing/selftests/lib/config
@@ -0,0 +1,3 @@
+CONFIG_TEST_PRINTF=m
+CONFIG_TEST_BITMAP=m
+CONFIG_PRIME_NUMBERS=m
-- 
2.14.1

